### PR TITLE
flake8 linting

### PR DESCRIPTION
**Pull request by anikaweinmann**
2     E222 multiple spaces after operator
1     E225 missing whitespace around operator
1     E261 at least two spaces before inline comment
2     E302 expected 2 blank lines, found 1
1     E502 the backslash is redundant between brackets
15    E722 do not use bare 'except'
2     E741 ambiguous variable name 'l'
2     F811 redefinition of unused 'DEFAULT_CONFIG_PATH' from line 34
9     F841 local variable 'e' is assigned to but never used
10    W503 line break before binary operator
29    F401 '.health_check.health_check' imported but unused

**Ignored:**
4     E402 module level import not at top of file 
7    F401 '.health_check.health_check' imported but unused

### CODE REVIEW

**Comment by mmacata** [comment]
Is this still a Draft? Or can I review? @anikaweinmann

**Comment by anikaweinmann** [comment]
@mmacata you can review

**Review by neteler — APPROVED** [review]
Nice work!

File: .flake8 [hunk 1]
```diff
@@ -33,3 +16,16 @@ exclude =
     ctypes,
     pydispatch,
     testsuite,
+
+per-file-ignores =
+    ./actinia_core/resources/common/app.py: E402, E501
```
Comment (mmacata) [inline .flake8:21]: I find it helpful to have a comment somewhere explaining the PEP8 codes. For actinia-module-plugin e.g. I added them on top: https://github.com/mundialis/actinia-module-plugin/blob/main/.flake8#L3

**Review by mmacata — CHANGES_REQUESTED** [review]
Great! Thank you for this huge effort! This improves the quality a lot :) 
I have some remarks though but I'm happy to see this in action.

File: src/actinia_core/testsuite.py [hunk 2]
```diff
@@ -37,9 +37,9 @@
 from .resources.common import redis_interface
 from .resources.common.config import global_config
 from .resources.common.user import ActiniaUser
-from .endpoints import create_endpoints
+# from .endpoints import create_endpoints
```
[the remark this answers is not in the document]
Comment (anikaweinmann) [reply src/actinia_core/testsuite.py:40]: Yes the tests are all running. I will remove the lines

File: .flake8 [hunk 3]
```diff
@@ -33,3 +16,16 @@ exclude =
...
+    ./actinia_core/testsuite.py: F401
+    ./actinia_core/resources/ephemeral_processing.py: F401, W605
+    ./actinia_core/resources/common/storage_interface_gcs.py: F401
+    ./actinia_core/main.py: F401
+    ./actinia_core/resources/common/process_queue.py: F401
+    ./actinia_core/resources/common/process_chain.py: W605
+    ./actinia_core/resources/common/list_grass_modules.py: F821
+    ./actinia_core/resources/common/aws_sentinel_interface.py: E501
+
+# ./actinia_core/resources/common/list_grass_modules.py: F821: undefined names after missing import PyQt4 (not used file?)
```
[the remark this answers is not in the document]
Comment (anikaweinmann) [reply .flake8:31]: The module XML is transfromed to a module list and I think it is written to the `grass_module_list.py`. I think it is only a script to create the file.

File: src/actinia_core/main.py [hunk 4]
```diff
@@ -41,8 +41,8 @@
 __maintainer__ = "Sören Gebbert"
 __email__ = "soerengebbert@googlemail.com"
 
-if os.environ.get('DEFAULT_CONFIG_PATH'):
-    DEFAULT_CONFIG_PATH = os.environ['DEFAULT_CONFIG_PATH']
+# if os.environ.get('DEFAULT_CONFIG_PATH'):
```
[the remark this answers is not in the document]
Comment (anikaweinmann) [reply src/actinia_core/main.py:44]: Yes it is possible

[the remark this answers is not in the document]
Comment (mmacata) [reply src/actinia_core/main.py:44]: cool

**Review by mmacata — APPROVED** [review]

**Pull request merged**